Sources/SwiftBuild/SWBBuildServiceSession.swift

```diff
@@ -12,7 +12,7 @@
 
 import Foundation
 
-import SWBProtocol
+public import SWBProtocol
```

Comment:
Not needed. (see https://github.com/swiftlang/swift-build/pull/760/files#r2312690176)